docs/src/development/troubleshoot.md

```diff
@@ -197,7 +205,7 @@ Invisible errors during the build and deploy phase can cause increased wait time
 ### Build and deploy hooks
 
 [`build` and `deploy` hooks](/configuration/app/hooks/_index.md) can cause long build times.
-If they run into issues, they can cause the build to fail or hang indefinitely.
+If they [run into issues](../configuration/app/app-reference.md#hook-failure), they can cause the build to fail or hang indefinitely.
```

Comment:
I don't think that the link actually adds information to the context here. It's mostly about how to cause them to fail, which is the opposite of helpful here. The issues there are separate from the ones here.

Reply:
That is very correct indeed, the link only adds more confusion! Thanks for spotting it, I removed it.